### DISCUSSION TITLE
varsome killer

### DISCUSSION THREAD

**Discussion by dridk** [vidboda/MobiDetails#28] [Ideas]
Hello,

Vous voulez pas faire un design à la google / varsome ? Une simple bar de recherche au centre ? Histoire de récupérer les utilisateurs de varsomes ? 

Sinon, il y a elastic app search ( version open source )  qui génère ce moteur de recherche avec la layout du design en react.  Il faut juste envoyer  le Json avec toutes la bdd et zoouuuu , c'est magique ! 
https://www.elastic.co/fr/enterprise-search/

**Comment by vidboda** [comment]
salut,

ça ferait pas de mal un petit rafraichissement (voire un gros) mais c'est pas vraiment dans mes priorités du moment. J'avoue que je ne vais pas sur varsome donc je vois pas trop leur design. Et le but c'est pas de copier mais bien de proposer autre chose!

**Reply by dridk** [reply]
En gros varsome a crée une dependance chez les biologistes avec un super outil gratuit au début mais qui suçe toutes nos données et méta données pour se faire  une super database qu'ils valoriseront plus tard.
 _" Vous voulez savoir si ce variant est déjà vu ? Payez un supplement .."_

En d'autres mots, mobidetails est le seul chanlenger viable proposant une alternative dans le domaine public ! L'utilisation de cet outil permettra de faire de grosse économie tout en gardant la maitrise de nos données... 
Donc vous avez une grosse responsabilité ! 
Je peux aider si besoin ! 

 _Au secours Mobidetail, vous êtes notre seul espoir_

**Reply by dridk** [reply]
Remarque  vous avez une API REST ? Ca devrait pas être difficile à faire

**Comment by vidboda** [comment]
Ah lala que de responsabilités!

pour l'api la réponse est oui:
https://mobidetails.iurc.montp.inserm.fr/MDAPI/

Si besoin d'intégrer un bouton sur une appli existante, je peux fournir un code js de 8-10 lignes qui fait le job

**Reply by dridk** [reply]
Je n'ai pas trouvé de endpoint sur l'API pour le "moteur de recherche". En gros , qd je tappe "T790M" sur le site, il me trouve tout seul toutes les infos de la mutation dans le gène EGFR etc ..

**Comment by vidboda** [comment]
ah oui exact il n'y a pas d'endpoint pour le moteur de recherche. C'est un vrai besoin?
